docs/src/modeling/forms/migration/README.md

```diff
@@ -0,0 +1,3 @@
+# Migration
+
+## Migration V1 -> V2
```

Comment:
Ich schätze mal, dass das hier ein Platzhalter für dein JSONSchema Konzept ist

Reply:
Ich ergänze hier mal schon alles, das wir wissen.
In den einzelnen Tickets muss die Seite dann noch angepasst werden